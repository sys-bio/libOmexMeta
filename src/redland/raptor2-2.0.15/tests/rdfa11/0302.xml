<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml">
    <head>
        <title>Test 0302: various types of tokens in @typeof</title>
    </head>
    <body>
        <div vocab="http://schema.org/" resource="http://openspring.net/scor#me"
             typeof="Person foaf:Person http://purl.org/dc/terms/Agent">
            <a property="homepage" href="http://openspring.net/">
                <span property="name">St√©phane Corlosquet</span>
            </a>
        </div>
    </body>
</html>