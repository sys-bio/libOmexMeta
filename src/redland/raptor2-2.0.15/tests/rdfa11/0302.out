<http://rdfa.info/test-suite/test-cases/xhtml1/rdfa1.0/0302.xml> <http://www.w3.org/ns/rdfa#usesVocabulary> <http://schema.org/> .
<http://openspring.net/scor#me> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://schema.org/Person> .
<http://openspring.net/scor#me> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://xmlns.com/foaf/0.1/Person> .
<http://openspring.net/scor#me> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://purl.org/dc/terms/Agent> .
<http://openspring.net/scor#me> <http://schema.org/name> "St\u00E9phane Corlosquet" .
<http://openspring.net/scor#me> <http://schema.org/homepage> <http://openspring.net/> .
